python-client/setup.py

```diff
@@ -2,79 +2,91 @@
 import re
 
 import pkg_resources
+from pathlib import Path
 from setuptools import setup, Command
+from setuptools.command.build import SubCommand
 from setuptools.command.build_py import build_py
 
 
-class GrpcTool(Command):
-    user_options = []
-    out_path = 'giskard/ml_worker/generated'
+class GrpcTool(Command, SubCommand):
+    user_options = [
+        ("build-lib=", "d", 'directory to "build" (copy) to'),
```

Comment:
do we actually need to parameterize it? I can't imagine when we'd want to use another destination directory

Reply:
See comment below